proto/osmosis/smartaccount/v1beta1/tx.proto

```diff
@@ -34,7 +34,7 @@ message MsgAddAuthenticatorResponse { bool success = 1; }
 // MsgRemoveAuthenticatorRequest defines the Msg/RemoveAuthenticator request
 // type.
 message MsgRemoveAuthenticator {
-  option (amino.name) = "osmosis/MsgRemoveAuthenticator";
+  option (amino.name) = "osmosis/smartaccount/remove-authenticator";
```

Comment:
did we verify name length on this? sometimes ledger can only handle certain lengths

Reply:
![Selection_063](https://github.com/user-attachments/assets/5c276dd8-45e7-414d-86eb-5b0ef37fa57b)

Reply:
I signed with a ledger to make sure this works as well => https://osmosis-network.slack.com/archives/C027ELA290B/p1722618905776139?thread_ts=1722618892.008519&cid=C027ELA290B